src/components/Breadcrumb/BreadcrumbItem/BreadcrumbItem.style.ts

```diff
@@ -12,7 +12,7 @@ export const breadcrumbItemStyles = ({ active }: RequiredProperties<StyleProps>)
 ) => css`
   display: flex;
   cursor: default;
-  color: ${active ? theme.utils.getColor('darkGray', 400) : theme.utils.getColor('lightGray', 400)};
+  color: ${active ? theme.utils.getColor('darkGray', 400) : theme.utils.getColor('lightGray', 600)};
```

Comment:
according to [example](https://www.figma.com/file/CVZxTXakKaWeuGu4qqi3zW/VideoHunter?node-id=2905%3A2), last lvl color for breadcrumbs is black.